GitHubRepositories.ps1

```diff
@@ -1368,18 +1520,19 @@ function Get-GitHubRepositoryLanguage
         If not supplied here, the DefaultNoStatus configuration property value will be used.
 
     .OUTPUTS
-        [PSCustomObject[]] List of languages for the specified repository.  The value shown
```

Comment:
.OUTPUTS is only expecting an object type to be specified, not a description. Check the `returnValues` object of the `MamlCommandHelpInfo` object that is returned by `Get-Help`.

Reply:
That's not what the [docs say though](https://docs.microsoft.com/en-us/powershell/scripting/developer/help/comment-based-help-keywords?view=powershell-7#outputs): 

```
.OUTPUTS
The .NET Framework type of the objects that the cmdlet returns. You can also include a description of the returned objects.
```